File: hw/toapanta/u2/hw19inherence/pythonFarmSystem/Pig.py
<<<<<<< HEAD
from FarmAnimal import FarmAnimal
class Pig(FarmAnimal):
    def __init__(self, id, breed, born_on, gender, is_able_to_reproduce, weight, cage):
        super().__init__(id, breed, born_on, gender, is_able_to_reproduce, weight, cage)

    def __str__(self):
=======
from FarmAnimal import FarmAnimal
class Pig(FarmAnimal):
    def __init__(self, id, breed, born_on, gender, is_able_to_reproduce, weight, cage):
        super().__init__(id, breed, born_on, gender, is_able_to_reproduce, weight, cage)

    def __str__(self):
>>>>>>> 203b676d8f105a34d549ec251a59c11aa2c57532
        return f"Pig{{, {super().__str__()}}}"

File: hw/toapanta/u2/hw19inherence/javaScriptFarmSystem/model/Pig.js
<<<<<<< HEAD
const FarmAnimal = require('./FarmAnimal');

class Pig extends FarmAnimal {
    constructor(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
    }

    toString() {
        return `Pig{, ${super.toString()}}`;
    }
}

=======
const FarmAnimal = require('./FarmAnimal');

class Pig extends FarmAnimal {
    constructor(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
    }

    toString() {
        return `Pig{, ${super.toString()}}`;
    }
}

>>>>>>> 203b676d8f105a34d549ec251a59c11aa2c57532
module.exports = Pig;